CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 Changes
 =======
 
+# 3.21.0 / 2023-07-03
+
+* [FEATURE] Trust new APT and RPM keys. ([#782][])
```

Comment:
The second `[]` seems to be a placeholder for the contributor's name, if it's not filled we might want to dismiss it?